### ISSUE TITLE
Set Recommended/Default Collateral Ratio to At Least 5 to Avoid Newbie Get Burned Easily

### ISSUE THREAD

**Issue by bangzi1001** [bitshares/bitshares-ui#1622]
**Is your feature request related to a problem? Please describe.**
Newbie who borrow bitCNY/bitUSD get burned easily because they are more likely to set 1.75 Collateral Ratio until they get margin call. Once they get margin call, they will either become conservative or stop borrow bitCNY/bitUSD. If they stop borrow bitCNY/bitUSD, then is not good for Bitshares.

**Describe the solution you'd like**
1. Set recommended/default Collateral Ratio to At Least 5. 
2. Description should encourage borrowers set Collateral Ratio of 5 and above.

**Comment by startailcoon** [comment]
I'd like to hear from others before we start changing the CR requirements. Today you get a warning when your ratio is +.5 from the assets MCR.

I can agree that we could show this error up to a higher number, making users aware of the risks involved because of a volatile market.

**Comment by wmbutler** [comment]
I agree with @fractalnode. Maybe a link to the help section with a practical explanation of how investors can use the ratios to their advantage.

**Comment by clockworkgr** [comment]
@sschiessl-bcp If spotlight makes a wizard of this, we can simply modify the copy on the standard borrowing pop-up to redirect newbies to the spotlight instead.
